### PR TITLE
[DEV APPROVED] 8154 new glossary terms

### PR DESCRIPTION
**Pull request by thecountgs**
Adding new Glossary terms for the live A/B test of this feature.

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1733)

### CODE REVIEW

**Comment by johnplayer1982** [comment]
:+1:

File: public/glossary.json [hunk 1]
```diff
@@ -2,21 +2,27 @@
   "glossary": [
     {
       "term": "annuity",
-      "description": "A type of retirement income that provides you with a regular payment, usually for life."
+      "description": "A retirement income product that guarantees a regular income for the rest of your life."
     },{
-      "term": "cash lump sum",
-      "description": "An amount of cash set by law that you can take at retirement free of tax. It's usually up to a quarter of your pension. Sometimes simply referred to as 'tax-free cash' or 'cash lump sum'."
+      "term": "lifetime annuity",
+      "description": "A retirement income product that guarantees a regular income for the rest of your life."
```
Comment (Tr4pSt3R) [inline public/glossary.json]: Is an _Annuity_ mean to have the same description as a _Lifetime Annuity_?

Comment (thecountgs) [inline public/glossary.json]: It does have the same meaning. per: https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx?acid=5D6AC326B4D272C23E446C28586BAF51#page=board/5624876525867731357&appConfig=eyJhY2lkIjoiREI4ODcwRjkxMDNDRTM2NTlBMzhDNTRBRTVBNUU1N0UifQ==&boardPopup=userstory/8154/silent

**Comment by Tr4pSt3R** [comment]
👍

**Pull request merged**